DigitalLearningSolutions.Data/Models/Logo.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace DigitalLearningSolutions.Data.Models
+{
+    using System;
+
+    public class Logo

```

Comment:
What about the logic to scale the image, is that no longer needed? 🙂

Reply:
The logic to scale the image needs the current native width and height of the image, which are not fields in the Brands table so to use the old logic we would either need to parse the JPEG to find these values. The approach I took was to let the image coming out of the LogoService be whatever size it likes, and control its size using CSS `max-height` and `max-width`, using the same values as what was in the previous code (width of `280px` and height of `75px`). Using this approach the image scaling logic isn't needed for it is done by the browser instead. Which approach do you prefer? 🙂

Reply:
Okay. As long as the centre logos look the same as they did before then I'm happy 🙂

Reply:
I've check 5 examples and they look the same, and I'm fairly sure its functionally equivalent 🙂